{{cookiecutter.repo_name}}/ci/templates/.travis.yml

```diff
@@ -1,9 +1,12 @@
-language: python
-python: '3.5'
 sudo: false
+language: python
+python: "3.4"
 env:
+{% if cookiecutter.c_extension_support|lower == "yes" %}
   global:
-    LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - SEGFAULT_SIGNALS="abrt segv"
```

Comment:
Honestly, in turn, I didn't know about `all`. :) However [the rest](http://stackoverflow.com/a/18947029) signals don't seem to be that common, IMHO:
  - `SIGBUS` is a HW fault, there's nothing we could do to fix it
  - `SIGSTKFLT` is "stack fault on coprocessor", unused on Linux
  - `SIGILL` and `SIGFPE` are more real-world though

What's about setting it to the explicit `SEGFAULT_SIGNALS="ill abrt fpe segv"`? Suggesting just because _explicit is better than implicit_.